examples/js/loaders/GLTFLoader.js

```diff
@@ -1573,10 +1587,12 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
-	GLTFParser.prototype.parse = function ( onLoad, onError ) {
+	GLTFParser.prototype.parse = function () {
 
 		var json = this.json;
 
+		var onLoad = ( this.options.onLoad !== undefined ) ? this.options.onLoad : function(){};
+
```

Comment:
Yes, currently `GLTFParser` is initialized only once, with `onLoad` method. This was just prevention. `onError` can be undefined, because it is passed as catch parameter, but undefined `onLoad` execution will thrown exception. I'll remove this checking